app/services/member_services/member_updater.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module MemberServices
+  class MemberUpdater
+    attr_reader :member
+    delegate :errors, to: :member
+
+    def initialize(email, params)
+      @email = email
+      @params = params
+    end
+
+    def run
+      @member = Member.find_by_email(@email)
```

Comment:
So I guess this PR is not to make it possible to update an email...?

Reply:
`email` is the identifier field used to find a member.
`params` is meant to include the values to be updated, it can include `email`.

Reply:
Ah I see, there are two email keys. One top level key for searching the member, and one in the `member` hash. Gotcha.